tests/rules/pow_metavar_comparison.py

```diff
@@ -0,0 +1,12 @@
+
+# ruleid: pow-metavar-comparison 
+x = foo(2, 3)
+
+# ruleid: pow-metavar-comparison 
+y = foo(2.0, 3)
+
+# ruleid: pow-metavar-comparison 
+w = foo(2, 3.0)
+
+# ruleid: pow-metavar-comparison 
+z = foo(2.0, 3.0)
```

Comment:
Add some examples where this doesn't match too, otherwise it's not a very strong test